blocks/video-player-block/features/video-player/default.jsx

```diff
@@ -31,7 +31,7 @@ const videoLayouts = {
 		hideVideoTitle,
 		title,
 	}) => (
-		<Stack className={`${BLOCK_CLASS_NAME}__inline`}>
+		<Stack className={`${BLOCK_CLASS_NAME} ${BLOCK_CLASS_NAME}__inline`}>
```

Comment:
Probably the one reason I dislike BEM... the repetitive/redundant class names to overcome issues that it creates.  (just ranting)